### PR TITLE
Document schema

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -66,7 +66,10 @@ lazy val root = (project in file("."))
     name := "akka-persistence-spanner-root",
     publishTo := Some(Resolver.file("Unused transient repository", file("target/unusedrepo")))
   )
-  .aggregate(docs, journal)
+  .aggregate(journal)
+
+lazy val dumpSchema = taskKey[Unit]("Dumps schema for docs")
+dumpSchema := (journal / runMain in (Test)).toTask(" akka.persistence.spanner.PrintSchema").value
```
Comment (johanandren) [inline build.sbt:72]: Such programmer seriousness. I just copied the thing..

File: journal/src/test/scala/akka/persistence/spanner/SpannerSpec.scala [hunk 2]
```diff
@@ -112,15 +111,14 @@ object SpannerSpec {
...
-        persistence_id STRING(MAX) NOT NULL,
-        deleted_to INT64 NOT NULL,
-      ) PRIMARY KEY (persistence_id)
-  """
+  def deleteMetadataTable(settings: SpannerSettings): String =
+    s"""CREATE TABLE ${settings.deletionsTable} (
+    persistence_id STRING(MAX) NOT NULL,
+    deleted_to INT64 NOT NULL,
+) PRIMARY KEY (persistence_id)
+"""
```
Comment (johanandren) [inline journal/src/test/scala/akka/persistence/spanner/SpannerSpec.scala:121]: funky indent?

Comment (chbatey) [reply journal/src/test/scala/akka/persistence/spanner/SpannerSpec.scala:121]: Makes it look nice in the docs

**Comment by chbatey** [comment]
I'll update this with snapshots too

**Pull request merged**